tests/helpers/permissions.ts

```diff
@@ -8,11 +8,12 @@ export const skipIfNotSupported = (lxdVersion: LxdVersions) =>
     "Fine grained authorisation is not available for lxd 5.0",
   );
 
-export const assertModificationStatus = async (
+export const assertTextValue = async (
   page: Page,
   statusText: string,
+  exact = false,
 ) => {
-  const status = page.getByText(statusText);
+  const status = page.getByText(statusText, { exact: exact });
   await expect(status).toBeVisible();
```

Comment:
As this is not about status anymore, I'd rename the variables:
statusText -> textValue
status -> textNode

Maybe a better name for the function is `assertTextVisible`, wdyt?

Reply:
I suppose the function is ascertaining the visibility of the textValue, as opposed to checking it's value against something else, so a rename does make sense.

Same for the properties, although out of curiosity, why the name "textNode" for the element hosting the textValue?

Reply:
I was thinking getByText would return a [dom node](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Introduction). Actually, it is returning a locator, so I suppose `textLocator` is a better name than `textNode`.

Reply:
```
export const assertTextVisible = async (
  page: Page,
  textValue: string,
  exact = false,
) => {
  const textNode = page.getByText(textValue, { exact: exact });
  await expect(textNode).toBeVisible();
};
```